global-script.lua

```diff
@@ -871,11 +891,13 @@ function DealPowerCards(deckZone, discardZone, clickFunctionName)
     end
 
     local xPadding = 4.4
-    local cardPlaceOffsetXs = {
-        -(2.5*xPadding)+1*xPadding,
-        -(2.5*xPadding)+2*xPadding,
-        -(2.5*xPadding)+3*xPadding,
-        -(2.5*xPadding)+4*xPadding,
+    local cardPlaceOffset = {
+        Vector(-(2.5*xPadding)+2*xPadding,0,0),
+        Vector(-(2.5*xPadding)+3*xPadding,0,0),
+        Vector(-(2.5*xPadding)+1*xPadding,0,0),
+        Vector(-(2.5*xPadding)+4*xPadding,0,0),
+        Vector(-(2.5*xPadding)+2*xPadding,0,6),
```

Comment:
how does adding the extra height interact with our current board setups? there are for sure going to be issues i feel with the large board PR you have going, just not sure if there are any problems with the current ones

Reply:
yea there are a handful of problematic setups. so instead of doing this with 2 rows, instead if you close the gap between the cards you can fit all 6 on one row https://gyazo.com/d90e4d6c08ed64f52adb9538ed11cba2